src/extensions/filter-control/utils.js

```diff
@@ -45,8 +59,14 @@ export function existOptionInSelectControl (selectControl, value) {
 }
 
 export function addOptionToSelectControl (selectControl, _value, text, selected) {
-  const value = (_value === undefined || _value === null) ? '' : _value.toString().trim()
-  const $selectControl = $(selectControl.get(selectControl.length - 1))
+  let value = (_value === undefined || _value === null) ? '' : _value.toString().trim()
+
+  value = value.replace(/(<([^>]+)>)/ig, '').replace(/&[#A-Za-z0-9]+;/gi, '').trim()
+
+  // Double check if the select control is a jquery object.
+  if (!(selectControl instanceof jQuery)) {
```

Comment:
Do we need that check ?
If selectControl isn't a instanceof jQuery then we have already failed beforehand, which means the select is broken i guess.